src/Imdb/Calendar.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+#############################################################################
+# IMDBPHP                              (c) Giorgos Giagas & Itzchak Rehberg #
+# written by Giorgos Giagas                                                 #
+# extended & maintained by Itzchak Rehberg <izzysoft AT qumran DOT org>     #
+# http://www.izzysoft.de/                                                   #
+# ------------------------------------------------------------------------- #
+# IMDBPHP TOP CHARTS                                      (c) Ricardo Silva #
+# written by Ricardo Silva (banzap) <banzap@gmail.com>                      #
+# http://www.ricardosilva.pt.tl/                                            #
+# ------------------------------------------------------------------------- #
+# IMDBPHP UPCOMMING RELEASES (based on IMDBPHP TOP CHARTS (c)Ricardo Silva) #
+# written/modified/extended by Ed (github user: duck7000)                   #
+# ------------------------------------------------------------------------- #
+# This program is free software; you can redistribute and/or modify it      #
+# under the terms of the GNU General Public License (see doc/LICENSE)       #
+#############################################################################
+
+namespace Imdb;
+
+/**
+ * Obtaining information about upcomming movie releases as seen on imdB
```

Comment:
```suggestion
 * Obtains information about upcoming movie releases as seen on IMDb
```

Reply:
Mm it must be late when i wrote this.. Changed it.